boards/arm/stm32/olimex-stm32-p407/Kconfig

```diff
@@ -7,7 +7,7 @@ if ARCH_BOARD_OLIMEX_STM32P407
 
 config STM32F4DISCO_OLIMEXP407_STACKSIZE
 	int "USB host waiter stack size"
-	default 1024
```

Comment:
It is inappropriate to increase the size of this tuned stack size.  Let's not do that.